src/pg_to_evalscript/javascript_processes/order.js

```diff
@@ -0,0 +1,46 @@
+function order(arguments) {
+  const { data, asc = true, nodata = null} = arguments;
+
+  if (data === null || data === undefined) {
```

Comment:
The parameters should probably be checked with the "new" `validateParameter` function.